sdk/src/main/java/ch/admin/bag/covidcertificate/sdk/android/data/AcceptedVaccineProvider.kt

```diff
@@ -8,20 +8,21 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-package ch.admin.bag.covidcertificate.eval.data
+package ch.admin.bag.covidcertificate.sdk.android.data
 
 import android.content.Context
-import ch.admin.bag.covidcertificate.eval.models.healthcert.eu.VaccinationEntry
-import ch.admin.bag.covidcertificate.eval.models.products.AcceptedVaccine
-import ch.admin.bag.covidcertificate.eval.models.products.Vaccine
-import ch.admin.bag.covidcertificate.eval.utils.SingletonHolder
+import ch.admin.bag.covidcertificate.sdk.core.models.healthcert.eu.VaccinationEntry
+import ch.admin.bag.covidcertificate.sdk.core.models.products.AcceptedVaccine
+import ch.admin.bag.covidcertificate.sdk.core.models.products.Vaccine
+import ch.admin.bag.covidcertificate.sdk.android.utils.SingletonHolder
 import com.squareup.moshi.JsonAdapter
 import com.squareup.moshi.Moshi
 import okio.buffer
 import okio.source
 import java.io.IOException
 
-class AcceptedVaccineProvider private constructor(context: Context) {
+class AcceptedVaccineProvider private constructor(context: Context) :
```

Comment:
We could name this `AndroidAcceptedVaccineProvider` to make it easier to distinguish the impl from the interface.